pkg/virt-handler/vm.go

```diff
@@ -2829,14 +2829,14 @@ func (d *VirtualMachineController) configureHousekeepingCgroup(vmi *v1.VirtualMa
 		return nil
 	}
 
-	hkcpu, err := strconv.Atoi(domain.Spec.CPUTune.EmulatorPin.CPUSet)
+	hkcpus, err := hardware.ParseCPUSetLine(domain.Spec.CPUTune.EmulatorPin.CPUSet, 100)
```

Comment:
nit: As we've discussed offline, please consider naming this magic number that has different values in various calls to `hardware.ParseCPUSetLine`.

Reply:
I suggest doing it in a separate PR for all references of `ParseCPUSetLine`